Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -73,6 +75,9 @@ public class NetworkManager : MonoBehaviour
         [FormerlySerializedAs("m_OnlineScene")]
         public string onlineScene = "";
 
+        public NetworkServer server;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/paulpach/MirrorNG/pullRequest?prid=4841467)